bugwarrior/config.py

```diff
@@ -150,28 +155,36 @@ def validate_config(config, main_section):
         get_service(service).validate_config(config, target)
 
 
-def load_config(main_section, interactive=False):
-    config = ConfigParser({'log.level': "INFO", 'log.file': None})
+def get_config_path():
+    """
+    Determine the path to the config file. This will return, in this order of
+    precedence:
+    - the value of $BUGWARRIORRC if set
+    - $XDG_CONFIG_HOME/bugwarrior/bugwarriorc if exists
+    - <dir>/bugwarrior/bugwarriorc if exists, for dir in $XDG_CONFIG_DIRS
+    - ~/.bugwarriorrc if exists
+    - $XDG_CONFIG_HOME/bugwarrior/bugwarriorc otherwise
+    """
```

Comment:
Good stuff. Could you correct the docs to reflect this? I'm thinking in particular of the first line of the "How to Configure" section which is now incorrect.